src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSource.java

```diff
@@ -346,7 +346,7 @@ private String createTempKafkaCertFile(String certData) {
         if (certData != null) {
             return createTempCertFile("kafka-cacert", certData);
         } else {
-            return null;
+            return "";
```

Comment:
Could you please add a comment to explain why this doesn't return `null`? You could mention your project, Camel or even the issue you faced.

Reply:
This is something that I'm still investigating. I went as far as Camel Kafka component creating params for Kafka URI: https://github.com/apache/camel/blob/007c4b6f7c054a01cf1426ea4ed32c7c0339ba71/components/camel-kafka/src/generated/java/org/apache/camel/component/kafka/KafkaEndpointUriFactory.java#L116